src/rdkafka_ssl.c

```diff
@@ -1114,6 +1165,32 @@ int rd_kafka_ssl_ctx_init (rd_kafka_t *rk, char *errstr, size_t errstr_size) {
         }
 #endif
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+        if(rk->rk_conf.ssl.openssl_engine_location && !rk->rk_conf.ssl.ptr_engine) {
```

Comment:
Code style is space before paren

Reply:
I don't think ssl.engine can be set here since this function is only called once.

Reply:
sure will add space in next iteration.

Here engine is just getting initialized. It is not loading the certificates. Since engine is supposed to be initialized only once so keeping it here. Let me know if you feel it should be moved to better place.